package/yast2-users.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Jul 16 13:13:26 UTC 2021 - David Diaz <dgonzalez@suse.com>
+
+- Use a more reliable way of knowing if editing a user (related to
```

Comment:
NP: I would reword it a bit: "Use a more reliable way of knowing if a user is being edited".

Reply:
Ok, done.